src/diskquota.h

```diff
@@ -75,6 +75,14 @@ extern int diskquota_worker_timeout;
 #define DiskquotaGetRelstorage(classForm) (0)
 #endif /* GP_VERSION_NUM */
 
+enum
+{
+	is_connected              = 1 << 0,
+	is_active_snapshot_pushed = 1 << 1,
+	is_abort                  = 1 << 2,
+	is_under_transaction      = 1 << 3,
```

Comment:
Other enum elements in diskquota.h are in uppercase. Let's use the same style

Reply:
uppered